core/dbt/events/core_types.proto

```diff
@@ -1809,12 +1809,23 @@ message ServingDocsExitInfoMsg {
     ServingDocsExitInfo data = 2;
 }
 
+message Group {
```

Comment:
I guess that you included everything that's in a group so that it's easy to do to_dict, but I don't think that we need things like resource_type, path, and original_file_path.  Probably not unique_id either, which is probably only used internally.  I'd just make a method on a Group to return a dictionary with the correct fields and remove the unnecessary bits.